webapp/pages/search/search-results.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <search-results :search="search" />
+</template>
+
+<script>
+import SearchResults from '~/components/_new/features/SearchResults/SearchResults'
+
+export default {
+  layout: 'default',
+  watchQuery: ['search'],
+  head() {
+    return {
+      title: this.$t('search.title'),
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-09-23T08:52:02Z" title="Wednesday, September 23rd 2020, 10:52:02 am +02:00">Sep 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,24 @@
+<template>
+  <search-results :search="search" />
+</template>
+
+<script>
+import SearchResults from '~/components/_new/features/SearchResults/SearchResults'
+
+export default {
+  layout: 'default',
+  watchQuery: ['search'],
+  head() {
+    return {
+      title: 'SearchResults',
```


`SearchResults` without a space and not translated … 😉